docs/contributing/pages/images.mdx

```diff
@@ -33,7 +33,7 @@ The image `docs/common-imgs/imgs/tags.png` is referenced in `docs/common-imgs/ta
 ![Tags](./img/tags.png)
 ```
 
-which is then included in many other files:
+which is then included in multiple files:
 
 ```markdown {filename:docs/platforms/ruby/common/enriching-events/tags/index.mdx}
 <Include name="common-imgs/tags" />
```

Comment:
Can't comment on the last line, but it needs to be updated to not reference `public` again.